frontend-ssr/components/cli_git/GitCli_CC.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { uuid } from '../../services/Utils';
+
+import styles from './GitCli.module.scss';
+
+interface Props {
+  commands: RegExp[] | string[],
+  successMessages: string[],
+}
+
+interface State {
+  currentIndex: number,
+  messages: Message[],
+  btnMessage: string,
+  loading: boolean,
```

Comment:
We don't need a separate value to see if we need to disable the form. We can do this based on `currentIndex`. I put a comment in the render method regarding this.